index.js

```diff
@@ -245,6 +246,13 @@ class Database {
     }
   }
 
+  /**
+   * Interrupts the database connection.
+   */
+  interrupt() {
```

Comment:
is this function actually useful in the sync world (since there's no way to call it really)? unless... users can define UDF to get back into JS? 🤔

Reply:
@zen0wu If your application uses Web Workers, you could call this, no? Not saying it's very common way to write apps, but in theory useful?